schema_cache.rst

```diff
@@ -0,0 +1,78 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+PostgREST uses the database schema cache to get information for several of its features:
+
+- For :ref:`resource_embedding`, it needs relationships' information, such as related tables, foreign keys, cardinalities, etc.
+- For :ref:`s_procs`, it needs access to their metadata, for example, parameters, return type or volatility. It also needs to verify if the function is overloaded.
+- To do an :ref:`upsert`, it looks for the primary keys.
+- To do an INSERT, it also looks for the primary keys in order to return the Location header.
+- For OPTION requests, it verifies the existence and the data modification capabilities of a table or view.

```

Comment:
Ditto for "data modification capabitilies". "instead of triggers" of the view would be more precise.